.vscodeignore

```diff
@@ -4,9 +4,12 @@ out/**
 node_modules/**
```

Comment:
I would actually prefer whitelisting. See [here](https://github.com/Microsoft/vscode-vsce/issues/12#issuecomment-285964473) how it is done.

Reply:
Didn't know that this was possible, but I now also prefer it :) Changed it.